src/main/java/Duke.java

```diff
@@ -1,10 +1,150 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println("    Hello from\n" + logo);
+        System.out.println("    _________________________________________");
+
+        System.out.println("    Hello! I'm Duke");
+        System.out.println("    What can I do for you?");
+        System.out.println("    _________________________________________");
+        System.out.println("     ");
+
+        //Read in input from user
+        String inputString;
+        Scanner in;
+
+        //Set up list to store user inputs
+        Todo[] tasks = new Todo[100];
+        int counter = 0;
+
+        //setup of exit flag
+        boolean exit = false;
+        while (!exit) {
+            in = new Scanner(System.in);
+            inputString = in.nextLine();
+
+            //switch cases for all specified input types
+            switch(inputString) {
```

Comment:
I like how you made use of indentations for the switch and case statements maximizing the readability.